distributed/scheduler.py

```diff
@@ -361,6 +447,8 @@ class WorkerState:
     _last_seen: double
     _local_directory: str
     _memory_limit: Py_ssize_t
+    _memory_other_history: "deque[tuple[float, Py_ssize_t]]"
```

Comment:
sadly, Cython doesn't seem to support ``from __future__ import annotations`` yet; it would have made the quotes unnecessary.

Reply:
I think it will in Cython 3 ( https://github.com/cython/cython/pull/3829 ). That's probably still a ways off though